dataflow/build.gradle

```diff
@@ -48,7 +48,7 @@ task liveVariableTest(dependsOn: compileTestJava, group: 'Verification') {
             if (!JavaVersion.current().java9Compatible) {
                 jvmArgs += "-Xbootclasspath/p:${configurations.javacJar.asPath}"
             }
-            classpath = sourceSets.test.compileClasspath
+            classpath = sourceSets.test.runtimeClasspath
```

Comment:
I guess these are leftovers that are already in master?

Reply:
No, this is a bug fix that was exposed by these changes.

Reply:
These fixes were already merged to master, but master had not been merged into this branch.  I fixed that.